models/user/index.js

```diff
@@ -521,7 +521,7 @@ class User extends Model {
   }
 
   static async updatePassword(username, newPassword, disableCode) {
-    const newHashedPassword = await account.hash(newPassword);
+    const newHashedPassword = await helpers.checkAndHashPassword(newPassword, [username]);
```

Comment:
What happens when checkAndHashPassword throws an error?

Reply:
Sure, that's a good catch. The request would hang, because not `res.json` nor `next` is called. I overlooked that. Have i not tested that? Shame on me. :)

Reply:
Then `updatePassword` will return a Promise which will be rejected. Nothing after calling the erroring function will be executed, like errors do.